pkg/tests/cli_test.go

```diff
@@ -90,14 +90,80 @@ func TestVerify(t *testing.T) {
 	tsrPath := getTimestamp(t, restapiURL, artifactContent, nonce, policyOID, tsrContainsCerts)
 
 	// write the cert chain to a PEM file
-	_, certChainPemPath := writeCertChainToPEMFiles(t, restapiURL)
+	pemFiles := writeCertChainToPEMFiles(t, restapiURL)
 
 	// It should verify timestamp successfully.
-	out := runCli(t, "--timestamp_server", restapiURL, "verify", "--timestamp", tsrPath, "--artifact", artifactPath, "--certificate-chain", certChainPemPath, "--nonce", nonce.String(), "--oid", policyOID.String(), "--common-name", commonName)
+	out := runCli(t, "--timestamp_server", restapiURL, "verify", "--timestamp", tsrPath, "--artifact", artifactPath, "--certificate-chain", pemFiles.certChainPath, "--nonce", nonce.String(), "--oid", policyOID.String(), "--common-name", commonName)
 	outputContains(t, out, "Successfully verified timestamp")
 }
 
-func TestVerifyPassLeafCertificate(t *testing.T) {
+func TestVerify_RootAndIntermediateCertificateFlags(t *testing.T) {
```

Comment:
I think we can rewrite all or most of these verify tests as table tests in a follow up PR.